tests/cmd-utils/wait-cronjob/main.go

```diff
@@ -51,10 +51,10 @@ func checkCronJobExists(clientset *kubernetes.Clientset) error {
 		cronjobs, err := clientset.BatchV1beta1().CronJobs(namespace).List(ctxWithTimeout, metav1.ListOptions{})
 		if err != nil {
 			if apierrors.IsNotFound(err) {
-				log.Debug("No cronjobs were found")
+				logrus.Debug("No cronjobs were found")
 				return false, nil
```

Comment:
I think you can remove this return, is enough to have the one on line 57

Reply:
Yes, thanks!